src/lightkurve/search.py

```diff
@@ -119,21 +155,33 @@ def __repr__(self, html=False):
         out = "SearchResult containing {} data products.".format(len(self.table))
         if len(self.table) == 0:
             return out
-        columns = [
-            "#",
-            "mission",
-            "year",
-            "author",
-            "exptime",
-            "target_name",
-            "distance",
-        ]
+        columns = REPR_COLUMNS_BASE
+        if self.display_extra_columns is not None:
+            columns = REPR_COLUMNS_BASE + self.display_extra_columns
+        # search_tesscut() has fewer columns, ensure we don't try to display columns that do not exist
+        columns = [c for c in columns if c in self.table.colnames]
+
         self.table["#"] = [idx for idx in range(len(self.table))]
         out += "\n\n" + "\n".join(self.table[columns].pformat(max_width=300, html=html))
         # Make sure author names show up as clickable links
         if html:
             for author, url in AUTHOR_LINKS.items():
                 out = out.replace(f">{author}<", f"><a href='{url}'>{author}</a><")
+            # special HTML formating for TESS proposal_id
+            tess_table = self.table[self.table["project"] == "TESS"]
+            if "proposal_id" in tess_table.colnames:
+                proposal_id_col = np.unique(tess_table["proposal_id"])
+            else:
+                proposal_id_col = []
+            for p_ids in proposal_id_col:
+                # for CDIPS products, proposal_id is a np MaskedConstant, not a string
+                if p_ids == "N/A" or (not isinstance(p_ids, str)):
+                    continue
+                # e.g., handle cases with multiple proposals, e.g.,  G12345_G67890
+                p_id_links = [f"""\
+<a href='https://heasarc.gsfc.nasa.gov/docs/tess/approved-programs.html#:~:text={p_id}'>{p_id}</a>\
```

Comment:
Yeah, `#:~:text` is a Chromium-only feature. I feel it's okay because for users using non-Chromium browsers, the link will still bring users to the page listing all proposals.